frontend/src/components/form/InterpreterSearch.tsx

```diff
@@ -8,20 +8,19 @@ import {
   StyledFormLabel,
   GridRow,
 } from 'components/form/inputs/DirectoryInputs';
-import AutocompleteInput from 'components/form/inputs/Autocomplete';
 import SearchDates from 'components/form/SearchDates';
 import { Schema, Initial } from 'components/form/schemas/search.schema';
 import Range from 'components/form/Range';
 import Check, { SingleCheck } from 'components/form/inputs/Check';
 import { StyledButton } from 'components/Buttons';
 
-import { languages } from 'constants/languages';
 import SearchContext from 'contexts/SearchContext';
 import { ErrorMessage, Formik, FormikProps, Form } from 'formik';
 import { withFlag } from 'components/reusable/withFlag';
-import { getLocations } from 'util/locationFetch';
+import { getLocations } from 'util/apiHelper';
 import { Location } from 'constants/interfaces';
 import { AutoCompleteField, ACFC } from './inputs/AutoCompleteField';
+import { AutoCompleteLanguage } from './inputs/AutocompleteLanguage';
```

Comment:
I'd really prefer if we could stick to absolute imports 👯

Reply:
good suggestion.